docassemble/StudentEvaluations/data/questions/massachusetts_educational_evaluations_request.yml

```diff
@@ -683,24 +715,53 @@ event: review_initial_request
 question: |
   Review your answers
 ---
+mandatory: True
+code: |  
+  email_sent_ok = al_user_bundle.send_email(editable=True, to=school.special_education_director_or_coordinator.email, template=notification)
+  email_sent_ok_1 = al_user_bundle.send_email(editable=True, to=requestor.email, template=notification)
+
+---
+template: notification
+subject: |
+  Greetings, ${school.special_education_director_or_coordinator.name.first } ${school.special_education_director_or_coordinator.name.last}
+content: |
+  This is an e-mail sent from Massachusetts Advocates for Children.
+
+  Have a nice day!
+
+---
 id: download initial_request
 event: initial_request_download
 question: |
   All done
-subquestion: |
-  Thank you. Your form is ready to download and deliver.
+# not sure if an if statement is needed for the email to send 
```

Comment:
I dont think this email method works. Quinten is working on adding a feature like this to the AL that is better. Lets remove it again